hypha/static_src/src/sass/apply/components/_form.scss

```diff
@@ -335,6 +335,39 @@
         }
     }
 
+    &__comments {
+        display: grid;
+        grid-template-areas: 'message' 'visibility' 'actions';
+        grid-template-rows: auto auto auto;
+        grid-template-columns: 1fr;
+
+
+        @include media-query(tablet-landscape) {
+            grid-template-areas: 'message visibility' 'actions actions';
+            grid-template-rows: auto auto;
+            grid-template-columns: 2fr 1fr;
+            column-gap: 2rem;
+
+            .wmd-input {
+                max-width: 100%;
+            }
+        }
+
+        .id_message {
+            grid-area: message;
+        }
+
+        .id_visibility_0 {
+            grid-area: visibility;
+        }
+
+        .button {
```

Comment:
@frjo  I was also looking at the same issue. These changes look better than mine. Just one thing, we can add `margin-top: 0;` in here, it will be more closer to input area as it is mentioned in issue comment.